src/ElvTenant.js

```diff
@@ -802,6 +802,106 @@ class ElvTenant {
 
     return res;
   }
+
+  async TenantFundUser({ asUrl, tenantId, userAddress, amount = 0.2}) {
+
+    console.log("TenantFundUser");
+    console.log(`as_url: ${asUrl}`);
+    console.log(`tenant_id: ${tenantId}`);
+    console.log(`user_address: ${userAddress}`);
+    console.log(`amount: ${amount}`);
+
+    // Initialize configuration
+    const config = {
+      configUrl: Config.networks[Config.net],
+      mainObjectId: Config.mainObjects[Config.net],
+    };
+
+    // Initialize EluvioLive and ElvAccount
+    const eluvioLive = new EluvioLive(config);
+    await eluvioLive.Init({
+      debugLogging: this.debug,
+      asUrl
+    });
+
+    const elvAccount = new ElvAccount({
+      configUrl: Config.networks[Config.net],
+      debugLogging: this.debug,
+    });
+    await elvAccount.Init({ privateKey: process.env.PRIVATE_KEY });
+
+    // check valid user address
+    if (!Utils.ValidAddress(userAddress)) {
+      throw Error(`Invalid user address provided: ${userAddress}`);
+    }
+    const usrAddr = Utils.FormatAddress(userAddress);
+
+    // check tenant root key
+    const tenantAddr = Utils.HashToAddress(tenantId);
```

Comment:
can all this logic be put into authd instead of duplicated here, just have a call to
/fund/:tid/:userAddress ?  or is that not possible?

Reply:
The api request accepts only the auth token created by tenant admins, it fails if we use any other key. This is to pre-check before sending the request.

Reply:
ah ok.  rather than duplicate the check code here, can we instead let it fail in the check there, fix the err msg we get back to make the problem clear?  or if there's something better about this, all good, was just asking.

just was commenting as otherwise changes to this process will have to be applied in two places.  not a big deal.

Reply:
removed the check in elv-live-js